Source/DafnyRuntime/DafnyRuntimeJava/src/test/java/dafny/EuclideanTest.java

```diff
@@ -236,4 +236,4 @@ void testBigIntegerMod() {
     assertEquals(BigInteger.ONE, DafnyEuclidean.EuclideanModulus(BigInteger.valueOf(13), two));
     assertEquals(BigInteger.ONE, DafnyEuclidean.EuclideanModulus(BigInteger.valueOf(-13), two));
   }
-}
+}
```

Comment:
nit: Nice to revert all these trailing newline changes to reduce the review noise.